service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -64,12 +64,17 @@ public Pipeline getPipeline(PipelinesEnum pipelineName) {
    * @param pipelineName - name of pipeline to update
    * @param workspaceProject - workspace project to update to
    * @param workspaceName - workspace name to update to
+   * @param workspaceStorageContainerUrl - workspace storage container URL to update to
    */
   public Pipeline updatePipelineWorkspace(
-      PipelinesEnum pipelineName, String workspaceProject, String workspaceName) {
+      PipelinesEnum pipelineName,
+      String workspaceProject,
+      String workspaceName,
+      String workspaceStorageContainerUrl) {
     Pipeline pipeline = getPipeline(pipelineName);
     pipeline.setWorkspaceProject(workspaceProject);
     pipeline.setWorkspaceName(workspaceName);
+    pipeline.setWorkspaceStorageContainerUrl(workspaceStorageContainerUrl);
     pipelinesRepository.save(pipeline);
```

Comment:
just double checking, because we require all the fields in the request body, none of the values passed in should be null right?  curious if for this we want to add the @NotNull and @Validated annotations since this is a super important operation for our service - https://www.baeldung.com/java-notnull-method-parameter#1-spring-boot-validation

Reply:
good idea, thanks!